wren-modeling-rs/core/src/mdl/builder.rs

```diff
@@ -10,6 +10,12 @@ pub struct ManifestBuilder {
     pub manifest: Manifest,
 }
 
+impl Default for ManifestBuilder {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
 impl ManifestBuilder {
     pub fn new() -> Self {
         Self {
```

Comment:
Why add a new impl `Default` for `default()`? Why not just make the `default()` into the impl `ManifestBuilder`?

Reply:
It's a standard trait in Rust. It's convenience for some design pattern.
https://rust-unofficial.github.io/patterns/idioms/default.html

Reply:
Thanks for your information.